recirq/qcqmc/hamiltonian.py

```diff
@@ -84,9 +84,6 @@ class HamiltonianFileParams(data.Params):
         n_orb: The number of spatial orbitals.
         n_elec: The total number of electrons.
         do_eri_restore: Whether to restore 8-fold symmetry to the integrals.
-        path_prefix: An optional path string to prepend to the default
```

Comment:
this is still an attribute / arg. Can you keep & update the docstring?

Reply:
Done.